temporal-opentracing/src/main/java/io/temporal/opentracing/provider/DataDogOpenTracingSpanBuilderProvider.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.opentracing.provider;
+
+import io.temporal.opentracing.SpanBuilderProvider;
+import io.temporal.opentracing.SpanCreationContext;
+import io.temporal.opentracing.internal.SpanBuilderFromSpanContentProvider;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This implementation of {@link SpanBuilderProvider} names and tags the OpenTracing spans in a way
+ * that is compatible with DataDog's APM tooling.
+ */
+public class DataDogOpenTracingSpanBuilderProvider extends SpanBuilderFromSpanContentProvider
```

Comment:
Can we rename it from DataDog into something more generic? It doesn't have to be used with DataDog, it doesn't depend on any DataDog classes or super custom logic? So, can we choose a prefix that describes the logic of this class instead of a specific vendor?

Reply:
Well, this is not just the span naming but also datadog-specific tag names.  I feel like somebody coming in here working with DataDog would benefit from having a provider that is known to work with that vendor.

Reply:
Discussed with @mfateev, let's leave DataDogOpenTracingSpanBuilderProvider name and datadog related constants inside this class.